pom.xml

```diff
@@ -17,7 +17,7 @@
 		<hibernate.version>4.1.0.Final</hibernate.version>
 		<junit.version>4.12</junit.version>
 		<hamcrest.version>1.3</hamcrest.version>
-		<keycloak.version>3.1.0.Final</keycloak.version>
+		<keycloak.version>2.5.5.Final-redhat-1</keycloak.version>
```

Comment:
Is the downgrade required to be compatible with currently used version of KC?

Reply:
yes, the newer client caused problems